src/tests/Contain.test.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+// import renderer from 'react-test-renderer';
```

Comment:
- [ ] please try to remove the unused commented code in order to keep your file clean and pro